src/main/webapp/testjsp/test.jsp

```diff
@@ -0,0 +1,36 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
```

Comment:
Perhaps we can be less liberal with abbreviation for tags when using tag libs, it's easier to read and since it's gonna be written in JSTL files, there's less need to conform to 110 chars